model/task/db.go

```diff
@@ -1282,3 +1285,22 @@ func AbortTasksForVersion(versionId string, taskIds []string, caller string) err
 	)
 	return err
 }
+
+func AddHostCreateDetails(taskId, hostId string, hostCreateError error) error {
+	if hostCreateError == nil {
```

Comment:
This makes us not persist the host details if there's no error doesn't it? Is that intended?

Reply:
Yeah, I called this HostCreateDetails instead of HostCreateErrors in case we ever want to add more details, but right now there's really no reason to have anything stored  if there's no error.